documentation/source/newsfragments/913.bugfix.rst

```diff
@@ -0,0 +1 @@
+Assigning Python ints to signals greater than 32-bit wide will now work correctly for negative values.
```

Comment:
Could you also add a second newsfragment of the type 'change' to list the change of behavior when assigning ints that are out of range of the signal? They used to truncate and warn, but now they throw.